Assets/HoloToolkit/Input/Scripts/Utilities/MicrophoneHelperSample.cs

```diff
@@ -0,0 +1,159 @@
+﻿
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// TODO: insert specific license information (MRTK is MIT)
+//
+// This sample code is a modified version of an official UWP sample which can be found on:
+// https://github.com/Microsoft/Windows-universal-samples/blob/master/Samples/SpeechRecognitionAndSynthesis/cs/AudioCapturePermissions.cs
+//
+//*********************************************************
```

Comment:
Take a look at the other headers in the toolkit.

```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Reply:
Updated!